engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -1038,14 +1048,14 @@ private void checkContainerStatus()
   private AllocateResponse sendContainerAskToRM(List<ContainerRequest> containerRequests, List<ContainerRequest> removedContainerRequests, List<ContainerId> releasedContainers) throws YarnException, IOException
   {
     if (removedContainerRequests.size() > 0) {
-      LOG.info(" Removing container request: " + removedContainerRequests);
+      LOG.info(" Removing container request: (size={}) {}", removedContainerRequests.size(), removedContainerRequests);
```

Comment:
This was old code and I just added the size info since printing out the array without size turned out to be missing a critical piece of info. Although I agree we don't need both this and the inside-loop logging.